packages/vite-plugin-cloudflare/src/runner-worker/index.ts

```diff
@@ -22,28 +22,27 @@ interface DurableObjectConstructor<T = unknown> {
 
 interface WorkflowEntrypointConstructor<T = unknown> {
 	new (
-		// Constructor type to be added in https://github.com/cloudflare/workerd/pull/3239
-		// ...args: ConstructorParameters<typeof WorkflowEntrypoint<T>>
-		ctx: ExecutionContext,
-		env: T
+		...args: ConstructorParameters<typeof WorkflowEntrypoint<T>>
 	): WorkflowEntrypoint<T>;
 }
 
+const IGNORED_KEYS = ["self", "tailStream"];
+
 const WORKER_ENTRYPOINT_KEYS = [
 	"fetch",
+	"queue",
 	"tail",
 	"trace",
 	"scheduled",
```

Comment:
NIT: If you are sorting alphabetically...?

Reply:
That was the intention but I see I still left `test` at the end. Will change.